cli/src/commands/commit.rs

```diff
@@ -40,7 +42,7 @@ pub(crate) struct CommitArgs {
     #[arg(long = "message", short, value_name = "MESSAGE")]
     message_paragraphs: Vec<String>,
     /// Put these paths in the first commit
-    #[arg(value_hint = clap::ValueHint::AnyPath)]
+    #[arg(add = ArgValueCandidates::new(complete::modified_files))]
```

Comment:
> Does it break dynamic completion?

No, doesn't look like it. I forgot to consider people who don't want to switch yet.

> The current dynamic completion on zsh is annoying

That's good to know. I'm assuming that's an issue with clap_complete, right?

Reply:
> That's good to know. I'm assuming that's an issue with clap_complete, right?

I suspect so because fish completion works pretty well with the same data set.